src/Patch-registry.md

```diff
@@ -0,0 +1,81 @@
+| i2c | testing | Finn | <https://github.com/Rust-for-Linux/linux/pull/946> | |
+| io_pgtable | testing | Lina | <https://github.com/Rust-for-Linux/linux/pull/952/commits/f476b2b052165a40eed0a8fcb86b56f794ee62d8> | |
+| io_resource | testing | Maciej | <https://github.com/Rust-for-Linux/linux/pull/682> | |
+| jiffies | testing | Maria | <https://github.com/Rust-for-Linux/linux/pull/982> | |
+| mm | | Alice, Wedson | <https://github.com/Darksonn/linux/commit/7ba95d4fc5a8442ef5eb428b64109116717f7e47> | |
+| napi | testing | Amélie | <https://github.com/Rust-for-Linux/linux/pull/1018> | |
+| net_device | RFC | Tomo | | <https://lore.kernel.org/netdev/20230613045326.3938283-1-fujita.tomonori@gmail.com/> |
+| null block | RFC | Andreas | | <https://lore.kernel.org/rust-for-linux/20230503090708.2524310-1-nmi@metaspace.dk/> |
+| pci | testing | Tomo | <https://github.com/Rust-for-Linux/linux/pull/856> | |
+| phy | review | Tomo | | <https://lore.kernel.org/rust-for-linux/20231026001050.1720612-1-fujita.tomonori@gmail.com/> | 
+| pm_runtime | testing | Maciej | <https://github.com/Rust-for-Linux/linux/pull/700> | |
+| puzzlefs | RFC | Ariel | | <https://lore.kernel.org/rust-for-linux/20230726164535.230515-1-amiculas@cisco.com/> |
+| ramfs | abandoned | Fox | <https://github.com/Rust-for-Linux/linux/pull/409> | |
+| rbtree | testing | | <https://github.com/Darksonn/linux/commit/edb94cbf99f6d35bca05e052e997542f07c085ab> | |
+| regulator/consumer | experimental | Fabien | <https://github.com/Rust-for-Linux/linux/pull/1040> | |
+| reset | testing | Nikos | <https://github.com/Rust-for-Linux/linux/pull/933> | |
+| rtkit (apple) | testing | Lina | <https://github.com/Rust-for-Linux/linux/pull/952/commits/f7708d02efab0d96e56f78d7e6dfa56adc948ef4> | |
+| scatterlist | RFC | Fox | | <https://lore.kernel.org/rust-for-linux/20230610104909.3202958-1-changxian.cqs@antgroup.com/> |
+| schedule | testing | Boqun | <https://github.com/Rust-for-Linux/linux/pull/861> | |
+| sk_buf | RFC | Tomo | | <https://lore.kernel.org/netdev/20230613045326.3938283-1-fujita.tomonori@gmail.com/> |
+| socket | RFC | Michele | | <https://lore.kernel.org/rust-for-linux/20230814092302.1903203-1-dallerivemichele@gmail.com/> |
+| spi | testing | Esteban | <https://github.com/Rust-for-Linux/linux/pull/264> | |
+| tarfs | testing | Wedson | <https://github.com/Rust-for-Linux/linux/pull/1037> | |
+| timer | testing | Maria | <https://github.com/Rust-for-Linux/linux/pull/982> | |
+| thread | testing | Boqun | <https://github.com/Rust-for-Linux/linux/pull/109> | |
+| usb | testing | Martin | <https://github.com/Rust-for-Linux/linux/pull/884> | |
+| virtio | testing | Richard | <https://github.com/Rust-for-Linux/linux/pull/886> | |
+| workqueue | merged | Alice | <https://elixir.bootlin.com/linux/v6.7-rc1/source/rust/kernel/workqueue.rs> | <https://lore.kernel.org/rust-for-linux/20230828104807.1581592-1-aliceryhl@google.com/> |
```

Comment:
For in-tree code, I think it's better to use link to Torvalds' tree: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/rust/kernel/workqueue.rs

Reply:
Changed that, thanks